api/openapi.yml

```diff
@@ -136,6 +202,10 @@ components:
           type: integer
         contact:
           type: boolean
+        oauth_provider:
+          type: string
+        oauth_id:
+          type: string
```

Comment:
Same OAuth issue here. We should not design around OAuth unless we are sure about it.